### PR TITLE
feat(backend/frontend/shared): update design of activities page tm-555

### PR DESCRIPTION
**Pull request by MappariChan**
image

### CODE REVIEW

File: packages/shared/src/modules/activities/libs/types/activity-counts.type.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 type ActivityCounts = {
 	commentCount: number;
+	isLikedByUser: null | number;
```
Comment (tarmiel) [inline packages/shared/src/modules/activities/libs/types/activity-counts.type.ts:3]: With is- prefix expected boolean

File: apps/backend/src/modules/activities/activity.repository.ts [hunk 2]
```diff
@@ -22,6 +22,16 @@ class ActivityRepository implements Repository<ActivityEntity> {
 		this.activityModel = activityModel;
 	}
 
+	private checkIsLikedByUserQuery(
```
Comment (tarmiel) [inline apps/backend/src/modules/activities/activity.repository.ts:25]: Such name supposes to return boolean

**Review by tarmiel — CHANGES_REQUESTED** [review]

**Review by ShadowOfTheSpace — APPROVED** [review]

**Review by Vfivew — APPROVED** [review]

File: packages/shared/src/modules/activities/libs/types/activity-response-dto.type.ts [hunk 3]
```diff
@@ -6,6 +6,7 @@ import { type ActivityPayloadMap } from "./activity-payload-map.type.js";
 type ActivityResponseDto<T extends ValueOf<typeof ActivityType>> = {
 	commentCount: number;
 	id: number;
+	isLikedByUser: boolean | null;
```
Comment (what1s1ove) [inline packages/shared/src/modules/activities/libs/types/activity-response-dto.type.ts:9]:
```suggestion
	isLikedByUser: boolean;
```
should be `boolean`. add `?? false` in cases where we need to handle it

File: apps/frontend/src/pages/feed/libs/components/feed-activity/feed-activity.tsx [hunk 4]
```diff
@@ -29,40 +34,60 @@ const FeedActivity: React.FC<Properties> = ({ activity }: Properties) => {
 		void dispatch(activitiesActions.likeActivity(activity.id));
 	}, [dispatch, activity.id]);
 
+	const likeButtonClasses = getValidClassNames(
+		styles["tool-button"],
+		(activity.isLikedByUser as boolean) && styles["liked"],
```
Comment (what1s1ove) [inline apps/frontend/src/pages/feed/libs/components/feed-activity/feed-activity.tsx:39]: but it is not a boolean

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Pull request merged**